src/pages/Login/Login.tsx

```diff
@@ -52,10 +52,8 @@ export const Login: React.FC = () => {
   return (
     <div className={styles.loginContainer}>
       <div className={styles.header}>
-        <Typography variant="extra-large-600" className={styles.text}>
-          Welcome to Opulus
-        </Typography>
-        <Typography variant="small-medium-400" className={styles.caption}>
+        <Typography variant={'h3'}>Welcome to Opulus</Typography>
+        <Typography variant={'p2'} color="#71717A">
```

Comment:
you don't have to wrap these in curly braces, just put in the string for the variant
```suggestion
        <Typography variant='p2' color="#71717A">
```